go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/rancher
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.5
```

Comment:
Do we need to add a `toolchain` statement here for a specific reason?

Reply:
It is automatically added by the `go mod tidy` command and seems to be the new "hard" requirement from Golang.